.circleci/config.yml

```diff
@@ -10,8 +10,18 @@ executors:
     resource_class: large
   appimage-executor:
     docker:
-      - image: openscad/appimage-x86_64-base:latest
+      - image: ${CIRCLE_PROJECT_USERNAME}/appimage-base:<< pipeline.git.revision >>
```

Comment:
Further down you use ${CIRCLE_SHA1} to get the revision. Should we use the same everywhere for clarity?